packer/vmware.json

```diff
@@ -0,0 +1,85 @@
+{
+  "builders": [
+    {
+      "boot_command": [
+        "<tab> text ks=http://{{ .HTTPIP }}:{{ .HTTPPort}}/{{user `ks_file`}}.cfg<enter>"
+      ],
+      "disk_size": 10240,
```

Comment:
This is correct actually.  I need to organize how those packer json files are used.  For the training VM, we need a bigger drive, since I'm adding 8GB of swap.  For this one 10GB total disk size is ok.